profile/plotter.go

```diff
@@ -0,0 +1,85 @@
+package profile
+
+import (
+	"encoding/csv"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/plotter"
+	"gonum.org/v1/plot/plotutil"
+	"gonum.org/v1/plot/vg"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// CSVPlotter plots every attribute as VM number increases
+func CSVPlotter(xStep int, filePath, inFile string) {
```

Comment:
a function name should be a verb, e.g., `PlotCVS`

Reply:
Done